### PR TITLE
OSDOCS-4417: Adds notes for 4.10.39

### PR DESCRIPTION
**Pull request by opayne1**
Adds notes for 4.10.39 

Version(s):
4.10 only

Issue:
[OSDOCS-4417](https://issues.redhat.com/browse/OSDOCS-4417)

Link to docs preview:
https://joealdinger.github.io/openshift-docs/RNs-4.10.39/release_notes/ocp-4-10-release-notes.html

QE review:
- [ ] QE has approved this change.
* QE not req for this change.

### CODE REVIEW

**Comment by opayne1** [comment]
Going to go ahead and put this on peer review since it ships today. I am having issues with my manual previews and I am not sure how to fix at the moment.
/label peer-review-needed

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -3546,4 +3546,30 @@ $ oc adm release info 4.10.38 --pullspecs
...
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-39"]
+=== RHSA-2022:7211 - {product-title} 4.10.39 bug fix and security update
+
+Issued: 2022-11-01
+
+{product-title} release 4.10.39, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:7211[RHSA-2022:7211] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7210[RHBA-2022:7210] advisory.
```
Comment (kcarmichael08) [inline release_notes/ocp-4-10-release-notes.adoc:3556]: Linked pages do not exist (I assume they aren't live yet)

**Review by kcarmichael08 — COMMENTED** [review]
This looks good to me! Just noting the links aren't active yet.

**Comment by kcarmichael08** [comment]
LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:3556]: Links are live now

**Pull request merged**